src/main/java/se/iths/crimedatabase/view/LoginPage.java

```diff
@@ -0,0 +1,26 @@
+package se.iths.crimedatabase.view;
+
+import org.springframework.security.authentication.AnonymousAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class LoginPage {
+
+    @GetMapping("/login")
+    public String showLoginPage() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        if (authentication == null || authentication instanceof AnonymousAuthenticationToken)
+            return "login";
+
+        return "redirect:/";
```

Comment:
We can move the error handling to the html document instead and just return "login" if we want